### ISSUE TITLE
Bug: EqMac Web Content occupies a lot of RAM

### ISSUE THREAD

**Issue by dpolivaev** [bitgapp/eqMac#729]
- [x] I have checked for a similar issue and sure it hasn't been reported before.

**Describe the bug**
I has to restart the previous version v 1.6.1 of EqMac after its component EqMac Web Content occupied 2.1 GB RAM. Now it occupies 139 MB but the amount of the occupied RAM is increasing over time. After writing this text the amount increased to 143 MB in less than 5 minutes.

<img width="132" alt="eqMacBug" src="https://user-images.githubusercontent.com/1558052/188969160-643e2e58-4ca2-436e-814d-9442da48bd97.png">

<img width="134" alt="eqMacBug2" src="https://user-images.githubusercontent.com/1558052/188969220-7f87f4bf-6cfe-4416-8929-5e131b375817.png">

**Setup information:**
 - Audio device used for playback: Monitor HP 28 4K HDR
  - Audio transmission interface: HDMI
 - macOS Version: 12.5.1
 - eqMac Version v1.6.2 and earlier
 - eqMac UI Version v4.2.1

**Comment by dpolivaev** [comment]
It seems to stop increasing now. I'll observe it and reopen it if needed.

**Comment by dpolivaev** [comment]
Now 172 MB

**Comment by nodeful** [comment]
Ok thanks, I will look into this ASAP

**Comment by dpolivaev** [comment]
I appreciate your attention. Just to let you know: Currently, the occupied memory has doubled up to ca 280 MB. I have not restarted it yet.

**Comment by nodeful** [comment]
if you can narrow down at which point and after what user behaviour does the memory consumption increase and doesn't go down it will help a lot. I tried to do many things with the app and sometimes the RAM usage would exceed 200MB but then always go back down to around 100. Do you use Expert EQ with Spectrum Analyzer?

**Comment by dpolivaev** [comment]
No, I don't use Expert EQ and any PRO features. I have restarted the app recently. Right after the start it uses around 140 MB.
I am attaching the screenshots with my current settings. I have disabled the OTA update option now, it was active when I saw 2.1 GB of RAM consumed.

<img width="194" alt="eqmacSettings1" src="https://user-images.githubusercontent.com/1558052/189483567-6f7d8db9-415b-48aa-b891-a774d519db9d.png">

<img width="195" alt="eqmacSettings2" src="https://user-images.githubusercontent.com/1558052/189483577-e0fb182f-c330-4fc2-b043-823e18a1b18f.png">

<img width="393" alt="eqmacCurrentRAM" src="https://user-images.githubusercontent.com/1558052/189483588-b50d799d-c025-4fa5-acd6-94b034614be3.png">

I am going to keep my eyes on it. If there is anything I could check let me know.
